src/components/Fetch/CollapseMenu.js

```diff
@@ -136,12 +141,56 @@ export class CollapseMenu extends React.PureComponent<Props> {
     }
   }
 
+  handleChangeStartRangeEvent = () => {
+    if (this.startRangeInput != null) {
+      this.props.changeStartRange(this.startRangeInput.valueAsNumber);
+    }
+  }
+
+  handleChangeEndRangeEvent = () => {
+    if (this.endRangeInput != null) {
+      this.props.changeEndRange(this.endRangeInput.valueAsNumber);
+    }
+  }
+
+  setStartRangeInputRef = (ref: ?HTMLInputElement) => {
+    this.startRangeInput = ref;
+  }
+
+  setEndRangeInputRef = (ref: ?HTMLInputElement) => {
+    this.endRangeInput = ref;
+  }
+
   render() {
     return (
       <Collapse className="collapse-menu" in={this.props.detailsOpen}>
         <div>
           <Form horizontal onSubmit={this.handleSubmit}>
             {showLogBox(this.props.logIdentity, this.setURLRef)}
+            <div className="range-div">
```

Comment:
Since this works together with the find box, I think it makes more sense to have the controls be to the right of the find button. What do you think about moving it there with a shorter label (maybe "From line ____ to ____")